### PR TITLE
163426476-setup test environment and configure Travis CI to run tests

### PR DESCRIPTION
**Pull request by ozone4real**
#### What does this PR do?
It sets up test environment and configures Travis CI to run tests.

### Description of Task to be completed?
- Set up environment for testing React/Redux.
- Configure Travis CI.

### CODE REVIEW

**Review by omenkish — DISMISSED** [review]

File: testSetup.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import register from 'ignore-styles';
+register(['.css', '.sass', '.scss']);
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```
Comment (p-funky) [inline testSetup.js]: Please put an extra line at the end of this file.

File: _test_/first.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import { shallow } from 'enzyme';
+import App from '../src/App';
+
+describe('first test', () => {
+  const wrapper = shallow(<App />);
+  it('should render the App', () => {
+    expect(wrapper.length).toEqual(1);
+    expect(wrapper.find('div')).toHaveLength;
+  });
+});
```
Comment (p-funky) [inline _test_/first.test.js:11]: Please put an extra line at the end of this file.

Comment (ozone4real) [reply _test_/first.test.js:11]: Okay. I have done this.

File: testSetup.js (hunk 1 above)
Comment (ozone4real) [reply testSetup.js]: Okay. I have done this.

**Review by EfeAgare — APPROVED** [review]

**Review by divinediscipline — APPROVED** [review]
Looks good to me

**Review by bobbylemm — APPROVED** [review]

**Pull request merged**